lib/inspec/plugin/v2/loader.rb

```diff
@@ -74,21 +91,67 @@ def activate(plugin_type, hook_name)
       # rubocop: enable Lint/RescueException
     end
 
-    def activate_mentioned_cli_plugins(cli_args = ARGV)
-      # Get a list of CLI plugin activation hooks
-      registry.find_activators(plugin_type: :cli_command).each do |act|
-        next if act.activated
-        # If there is anything in the CLI args with the same name, activate it
-        # If the word 'help' appears in the first position, load all CLI plugins
-        if cli_args.include?(act.activator_name.to_s) || cli_args[0] == 'help'
-          activate(:cli_command, act.activator_name)
-          act.implementation_class.register_with_thor
-        end
-      end
+    def plugin_gem_path
+      self.class.plugin_gem_path
+    end
+
+    def self.plugin_gem_path
+      # I can't beleive there isn't a simpler way of getting this
```

Comment:
I took a stab at it...me either...this seems like the simplest way...

Also, s/beleive/believe/